Governance Polls 2024-11-04 (#2782)
makerdao/community: governance/polls/Recentering Maker Brand - Novemeber 4, 2024.md

---
title: Recentering Maker Brand - November 4, 2024
summary: Signal your support or opposition to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.
discussion_link: https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Keep the Sky brand as the backend protocol brand
   2: Recenter the Maker brand
   3: Recenter the Maker brand with a limited brand refresh
start_date: 2024-11-04T16:00:00
end_date: 2024-11-07T16:00:00
---
# Poll: Recentering Maker Brand - November 4, 2024

> The Governance Facilitators have placed a Governance Poll into the voting system in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://github.com/makerdao/next-gen-atlas/blob/0ad10bf7adb809299f2e35ca048b62708a3bbb6c/Sky%20Atlas/Sky%20Atlas.html#L8697BD). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.

Reviewer: ```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```

> The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.

Reviewer: ```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Accessibility Facilitators in accordance with Atlas article [A.5.1.1.3 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, November 4 at 16:00 UTC.
```

**This is a plurality vote.**

- **You may vote for a single option.**
- **You should vote for the option that you prefer.**
- **If you would accept any of the options you should vote 'Abstain'.**

## Review

The community can vote in this poll to express support or opposition to either one or none of the following options:

- Option 1 - Keep the Sky brand as the backend protocol brand.
- Option 2 - Recenter the Maker brand.
- Option 3 - Recenter the Maker brand with a limited brand refresh.

> Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.2 - Brand Identity - Recentering Maker Brand Polls](https://github.com/makerdao/next-gen-atlas/blob/0ad10bf7adb809299f2e35ca048b62708a3bbb6c/Sky%20Atlas/Sky%20Atlas.html#L8697BD) to help inform your position before voting.

Reviewer: ```suggestion
Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.1.1.3 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403) to help inform your position before voting.
```

> **Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. In order to minimize confusion, this vote will have no winner displayed on the Governance Portal; however, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**

Reviewer: ```suggestion
**Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. However, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**
```

---
title: Recentering Maker Brand - November 4, 2024

> summary: Signal your support or opposition to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.

Reviewer 2: Support or opposition is not accurate, action required. Suggestion:

Signal your support of the following branding options to...

> summary: Signal your support or opposition to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.

Reviewer: ```suggestion
summary: Signal your support of the following branding options to 1) keep the Sky brand as the backend protocol brand; or 2) recenter the Maker brand; or 3) recenter the Maker brand with a limited brand refresh.
```

Please review the discussion [thread](https://forum.sky.money/t/weekly-atlas-edit-proposal-week-of-2024-10-28-0/25411) and the originating Atlas article [A.5.1.1.3 - Brand Identity - Recentering Maker Brand Polls](https://sky-atlas.powerhouse.io/#A.5.1.1.3_Recentering_Maker_Brand_Polls-133f2ff0-8d73-80ba-add3-d43c59ec407b%7C8d5a16c0e403) to help inform your position before voting.

**Note that the Governance Portal does not currently support the specified winning criteria (described under **Outcome**) for this poll. However, the winning criteria will still apply at the end of it and the winner will be acknowledged in a subsequent forum post by the Governance Facilitators.**

## Outcomes

**The winning criteria for this poll are as follows:**

- The option with the most votes will be selected as the winning choice.
- However, if Option 1 has the most votes individually, but the combined votes for Options 2 and 3 exceed those for Option 1, Option 2 will be declared the winner.

> 

Reviewer: ```suggestion
- The winning option will direct the Maker/Sky workforce, with any required Atlas Edits being presented at a later date.

```
